types/redis/index.d.ts

```diff
@@ -1179,7 +1180,7 @@ export interface Commands<R> {
 
 export const RedisClient: new (options: ClientOpts) => RedisClient;
 
-export interface RedisClient extends Commands<boolean>, EventEmitter {
```

Comment:
Ehhh, `any` is scary and unsafe; I'd rather you use `unknown` and add `// TypeScript Version: 3.0` to the header, if I'm honest.